docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '3.2'
 services:
   db:
-    image: postgres:10
+    image: postgres:11
```

Comment:
Let's change this back to 10, it really does need to match the version in the PaaS config. Per Slack, I think it works as long as you don't accidentally upgrade to 11 first :-)